recipes/indicators/all/conanfile.py

```diff
@@ -20,6 +21,14 @@ def configure(self):
         if self.settings.compiler.cppstd:
             tools.check_min_cppstd(self, 11)
 
+        if tools.Version(self.version) < "2.0" and self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration(
+                "dbg-mcro < 2.0 can't be used by {0} {1}".format(
```

Comment:
```suggestion
                "indicators < 2.0 can't be used by {0} {1}".format(
```

Or you can add a patch in 1.9: https://github.com/p-ranav/indicators/commit/74c36cbb9384345bb381cd55ea2955c16f952542#diff-6bfd146dc2629076b5df08b19b43ebb0aa24293d3fe0c0f580f14b265cb2edd6

Reply:
I don't want to patch the libraries if it doesn't give a lot of benefits. In this case, it doesn't, because it's easy to use version 2.0, which supports even gcc 4.9.